tests/requirements.txt

```diff
@@ -6,4 +6,4 @@ parameterized==0.7.4
 pandas==0.25.3; python_version < '3.9'
 networkx==2.5
 simplejson==3.16.0
-pymongo==3.7.1; python_version < '3.9'
+pymongo==3.7.1; python_version < '3.9'
```

Comment:
not sure why this file was touched?

Reply:
I made sure to add/commit only changes I need. I don't know what happened that this file got changed. It is a also not obvious what has changed both of these look the same. I'll resolve the conflicts.

Reply:
You should be able to restore it with `git checkout main -- tests/requirements.txt`

My guess is that the difference is a newline.